package-configuration/linux-cli/get-cloudify.py

```diff
@@ -130,6 +139,30 @@ def run(cmd, sudo=False):
     return proc
 
 
+def is_root():
+    """Check if running as root
+    """
+    return True if os.getuid() == 0 else False
+
+
+def drop_root_privileges():
+    """Drop root privileges
+
+    This is used so that when installing cloudify within a virtualenv
+    using sudo, the default behavior will not be to install using sudo
+    as a virtualenv is created especially so that users don't have to
+    install in the system Python or using a Sudoer.
+    """
+    # maybe we're not root
+    if not is_root():
+        return
+
+    if VERBOSE:
+        prn('Dropping root permissions...')
+    os.setegid(int(os.getenv("SUDO_GID")))
+    os.seteuid(int(os.getenv("SUDO_UID")))
+
```

Comment:
this will fail if user is running as root user (without sudo). instead might be better to:
```
os.seteuid(int(os.environ.get('SUDO_UID', 0)))
```